cdg-pitest-license.txt

```diff
@@ -0,0 +1,7 @@
+#License file for pitest extensions
```

Comment:
To be sure: I'd also revert the change to this file. It doesn't follow our standards anyway (missing dot at end of sentence, auto-generated timestamp), so I'd rather keep it exactly as provided.

Reply:
Or we should replace all comments with something like:
```
# Arcmutate license for Error Prone Support, requested by sending an email to
# support@arcmutate.com.
```
(Assuming the plugin doesn't care about the comments :eyes:)